src/Handlers/Events/NetworkTest.ts

```diff
@@ -1,8 +1,23 @@
 import { Event } from '../../Bus';
 
+export enum NetworkTestType {
+  PING = 'ping',
+  TRACEROUTE = 'traceroute'
+}
+export interface TracerouteTestConfig {
```

Comment:
A blank line is missed

```suggestion

export interface TracerouteTestConfig {
```

Reply:
done